dali/ft/ftbase.cpp

```diff
@@ -556,18 +558,21 @@ MemoryBuffer & OutputProgress::deserializeCore(MemoryBuffer & in)
     return in;
 }
 
-MemoryBuffer & OutputProgress::deserializeExtra(MemoryBuffer & in, unsigned version)
+MemoryBuffer & OutputProgress::deserializeExtra(MemoryBuffer & in, unsigned & version)
```

Comment:
I'm not sure you gain much from making the version a reference parameter.  The saving is minimal, and it makes the code harder to understand.